docs

```diff
@@ -0,0 +1 @@
+docs-archive
```

Comment:
Do we need this?

Reply:
This is good for testing directly in the `airflow-site-archive`. This is really symbolic linl (it just shown like that in GH UI). 

When you try to run `python -m http.server` for example in the main folder of `apache-airflow-archive` - you will get a "somewhat" browseable index to all the providers, but without that symbolic link, what you get is that it will leead to 404 (you can remove the link and see yourself).

Possibly we can also fix it in the way that we rename the `docs-archive` to `docs` - I thought we might want to do it as one of next steps - but I wanted to avoid it now - as it means few million files moved and likely again the need to rewrite the whole repo.